docs/about.md

```diff
@@ -5,21 +5,54 @@ hide:
 
 # Data Science in NHS England
 
-We are the NHS England Data Science Team with a vision: 
+<div markdown>
+
+![Image title](../images/DS_team_photo.jpeg){ width="450" alt-tex="Picture of the Data Science team stood on some steps in London."  align=right }
+
+
+We are the **[NHS England](https://www.england.nhs.uk/) Data Science Team**. 
```

Comment:
![image](https://github.com/nhsengland/datascience/assets/114224821/a675d6f4-0afb-46da-91fc-154226b5cd52)
im not in love with how the difference in boldness looks